### PR TITLE
Enhancement: KCLVM python package for PyPI

### PR DESCRIPTION
**Pull request by He1pa**
#46 add set_up.py to build KCLVM python package,  and upload it to PyPI.
PyPI project: https://pypi.org/project/kclvm/

### CODE REVIEW

**Review by Peefy — DISMISSED** [review]
LGTM

File: internal/setup.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    description='KCLVM', 
+    long_description='''A constraint-based record & functional language mainly used in configuration and policy scenarios.''',
+    author_email='',  
+    url='https://kusionstack.io/',  
+    packages=setuptools.find_packages(),
+
+    zip_safe=True,
+    # 依赖包
+    install_requires=[
```
Comment (Peefy) [inline internal/setup.py:28]: Can read from the `requirements.txt` file.

File: internal/setup.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    name='kclvm',
+    author='KCL Authors',
+    version='0.4.2.2',
+    license='Apache License 2.0',
+
+    description='KCLVM', 
+    long_description='''A constraint-based record & functional language mainly used in configuration and policy scenarios.''',
+    author_email='',  
+    url='https://kusionstack.io/',  
+    packages=setuptools.find_packages(),
```
Comment (Peefy) [inline internal/setup.py:24]: There is no dependency on some resource files added: such as `checksum.txt`.

Comment (nuaazz) [reply internal/setup.py:35]: Done.  Packing files are controlled by the MANIFEST.in  file, now the package including the entire kclvm directory.

File: internal/setup.py (hunk 1 above)
Comment (He1pa) [reply internal/setup.py]: Done

**Comment by He1pa** [comment]
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y fix #46


#### 2. What is the scope of this PR (e.g. component or file name):


add:
+ internal/MANIFEST.in
+ internal/setup.py
+ empty '\_\_init\_\_.py' files in dir: kclvm_py

modify:
+ scripts/build-kclvm/requirements.release.txt
+ internal/kclvm_py/scripts/requirements.txt

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):


- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

Use setuptools and twine to build KCLVM python package, and upload it to PyPI.
steps:
1. cd internal
2. modify kclvm version in setup.py
3. run `python setup.py sdist` to build package
4. run `twine upload dist/kclvm-0.x.x` to upload package to PyPI
5. input username and password of PyPI

PyPI project: https://pypi.org/project/kclvm/



#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 



#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:


- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] Other
steps:
1. cd internal
2. modify kclvm version in setup.py
3. run `python setup.py sdist` to build package
4. run `twine upload dist/kclvm-0.x.x` to upload package to PyPI
5. input username and password of PyPI



#### 6. Release note



Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

File: internal/setup.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# 5. input username and password of PyPI
+
+install_requires = []
+require_path = pathlib.Path(__file__).parent.joinpath("kclvm/scripts/requirements.txt")
+with open(require_path) as f:
+    requires = f.read().split('\n')
+    for require in requires:
+        install_requires.append(require)
+
+setup(
```
Comment (Peefy) [inline internal/setup.py:25]: file loss: /kclvm/api/version/checksum.txt

**Comment by He1pa** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by Peefy — APPROVED** [review]
LGTM

**Review by chai2010 — APPROVED** [review]
LGTM

**Pull request merged**